### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by karancode**
Fixes the install commands in install readme.

### CODE REVIEW

File: docs/getting_started/install.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ The `kops` binary is also available via our [releases](https://github.com/kubern
 
 ```shell
 curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
-chmod +x kops-linux-amd64
-sudo mv kops-linux-amd64 /usr/local/bin/kops
+chmod +x kops
+sudo mv kops /usr/local/bin/kops
```
Comment (hakman) [inline docs/getting_started/install.md:22]: Why this change? The linux binary is named `kops-linux-amd64`?

Comment (karancode) [reply docs/getting_started/install.md:22]: @hakman thanks. I just realized that it is because of the way it being download in here & in github readme docs.
`curl -LO` vs `curl -Lo kops`. 

So here, it should be `kops-linux-amd64` same are remote binary name.

(do you think we should unify it in both places?)

File: docs/install.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ From Github:
 
 ```bash
 curl -Lo kops https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-darwin-amd64
-chmod +x ./kops
-sudo mv ./kops /usr/local/bin/
+chmod +x ./kops-darwin-amd64
+sudo mv ./kops-darwin-amd64 /usr/local/bin/kops
```
Comment (karancode) [inline docs/install.md:18]: this should be `kops` here.

File: docs/getting_started/install.md (hunk 1 above)
Comment (hakman) [reply docs/getting_started/install.md:22]: Let's keep the `-LO` version for now in both places. Thanks for finding this.

Comment (hakman) [reply docs/getting_started/install.md:22]: Actually I looked in the file and there is `curl -Lo kubectl` also. So let's go with `-Lo kops` to keep same style.

Comment (karancode) [reply docs/getting_started/install.md:22]: thanks. I will make it uniform across. Could you PTAL e644b530725caefbf96107da67d79ec601105ad2

File: docs/install.md (hunk 2 above)
Comment (karancode) [reply docs/install.md]: fixed e644b530725caefbf96107da67d79ec601105ad2

File: docs/getting_started/install.md (hunk 1 above)
Comment (hakman) [reply docs/getting_started/install.md:22]: Thanks. Looks ok. Please squash also the changes.

Comment (karancode) [reply docs/getting_started/install.md:22]: done. thanks.

**Comment by hakman** [comment]
Thanks for your contribution @karancode.
/lgtm
/approve
